src/library-authoring/components/ComponentCard.tsx

```diff
@@ -27,6 +29,7 @@ type ComponentCardProps = {
 
 export const ComponentMenu = ({ usageKey }: { usageKey: string }) => {
   const intl = useIntl();
+  const editUrl = getEditUrl(usageKey);
```

Comment:
This is just a note that this was crashing, as I have Collections on my component list (we have an open PR handling the Collections results)

The following snippet made it work:
```typescript
const editUrl = usageKey && getEditUrl(usageKey);
```

Reply:
Thanks - fixed in cdd2b63f70c4dfa47faa9396ecdd8df62567a446